src/RTClib.h

```diff
@@ -78,6 +79,7 @@ class TimeSpan;
     inclusive.
 */
 /**************************************************************************/
+static TwoWire *RTCWireBus;
```

Comment:
This only allows a single Wire instance in use. Someone might expect to call begin() with a Wire, then begin() for a different RTC with another hardware bus or soft I2C etc. and be surprised when things don't work.
It'd be more useful to have it as a protected member of each RTC class.

Reply:
thanks @drak7 
you're absolutely right.
I'll get to it later today or tomorrow :)